web/themes/custom/fmcg_material/css/stylec.css

```diff
@@ -0,0 +1,6004 @@
+@charset "UTF-8";
```

Comment:
This looks like a mistype to me (the file name not the line)

Reply:
Ah yes that's correct!